cron/getIncrementBalance.js

```diff
@@ -0,0 +1,46 @@
+const cron = require('node-cron');
+const { familyModel, userModel, transactionModel } = require('../database/models');
+
+const { getLogger } = require('../helpers');
+const logger = getLogger('getIncrementBalance');
+
+async function main() {
+  try {
+    await cron.schedule('0 0 1 * *', async function () {
+
+      const allFamilies = await familyModel.find({});
+
+      await Promise.all(allFamilies.map(async item => {
+        await item.update(
+          {
+            $inc: {
```

Comment:
And with following approach $inc won't work. Inc adds some value to existing field. It will add calculated amount to previous `giftsForUnpacking` field value